extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -239,6 +272,14 @@ const renderInlineUi = (pageEl, attachmentEl) => {
     chipEl.prepend(makeImgElWithBG(src));
   }
 
+  Services.storyStoreServiceForOrNull(getWin(openAttachmentEl)).then(
```

Comment:
To summarize what we went over before the long weekend:
1. This needs to get the text before rendering to the dom. This will likely need to return a promise and render in amp-story-page when that resolves.
2. This logic should only run if attachmentEl is an amp-story-shopping-attachment.